client/app/visualizations/variables.less

```diff
@@ -0,0 +1,35 @@
+/* --------------------------------------------------------
+    Colors
+-----------------------------------------------------------*/
+@black: #000;
+@blue: #2196f3;
+@green: #4caf50;
+@orange: #ff9800;
+@redash-gray: rgba(102, 136, 153, 1);
```

Comment:
Since all these variables will be independent from main Redash styles (and later we're going to redefine them for Redash) - I think we should avoid using too generic names and names from Redash styles. Instead, let's prefix all variables with some namespace to clearly see that they are related to visualizations and avoid names conflicts in the future. For example, this variable may be called `@visualizations-gray: rgba(102, 136, 153, 1)`, and later, when using library in Redash - we'll define it as `@visualizations-gray: @redash-gray`. Same for others.